todo.js

```diff
@@ -0,0 +1,130 @@
+
+//DEFINITIONS
+var bullet = "- ";
+
+var button = $("<button>").attr("type", "submit").text("submit");
+var input = $("<input>").attr("type", "string");
+var label = $("<label>").append("Add an item <br>").append(this.input);
+
+var icon = function(src) {
```

Comment:
Looks like the `src` parameter always has the same value. why should it be pass in instead of having it in the constructor?